NAMESPACE

```diff
@@ -32,6 +35,9 @@ export(to_sonarqube)
 export(value)
 export(zero_coverage)
 import(methods)
+importFrom(httr,RETRY)
+importFrom(httr,content)
+importFrom(httr,upload_file)
```

Comment:
Required to use testthat mocking https://testthat.r-lib.org/reference/local_mocked_bindings.html?q=local_mock#namespaced-calls